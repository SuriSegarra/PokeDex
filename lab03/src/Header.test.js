import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.js';

test('renders App.js correctly', () => {
    const app = renderer
      .create(<Header />)
      .toJSON();
    expect(app).toMatchSnapshot();
  });