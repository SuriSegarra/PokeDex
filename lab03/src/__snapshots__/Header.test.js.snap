// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div>
  <section>
    <h1>
      <a
        href="https://fontmeme.com/pokemon-font/"
      >
        <img
          alt="pokemon-font"
          border="0"
          src="https://fontmeme.com/permalink/200213/54094a75ad0cab6a2805fe58ea555d47.png"
        />
      </a>
    </h1>
  </section>
</div>
`;
